source/MainMenuState.hx

```diff
@@ -251,7 +230,7 @@ class MainMenuState extends MusicBeatState
 					});
 				}
 			}
-			#if desktop
+			#if MODS_ALLOWED
```

Comment:
Uh, it seems kinda dumb to do this, what if you want to edit characters quickly?????

Reply:
You are right